plugins/services/src/js/components/LogView.js

```diff
@@ -0,0 +1,341 @@
+import React from 'react';
```

Comment:
Most parts of this file are extracted from `MesosLogView`. There will be a follow-up task to extract the remaining parts to `TaskFileViewer` and make it use `LogView` instead, so we can remove `MesosLogView`.